app/imports/api/sprints/methods.js

```diff
@@ -14,8 +14,16 @@ export const addSprint = new ValidatedMethod({
     validate: AddSprintSchema.validator({ clean: true }),
     run({ name, projectId }) {
         const userId = Meteor.userId();
+        const isOpenSprint = Sprints.find({ closed: false, projectId }).fetch().length;
+
         if (isProjectModerator(projectId, userId)) {
-            return Sprints.insert({ name, projectId });
+            if (!isOpenSprint) {
```

Comment:
You can also close other sprints when you create new one. This way is more user friendly.